src/commands/createTransactionCastVotes.js

```diff
@@ -88,18 +89,20 @@ export const actionCreator = vorpal => async ({ options }) => {
 
 	const allVotes = [...prependedVotes, ...prependedUnvotes];
 
-	return getInputsFromSources(vorpal, {
-		passphrase: {
-			source: passphraseSource,
-			repeatPrompt: true,
-		},
-		secondPassphrase: !secondPassphraseSource
-			? null
-			: {
-					source: secondPassphraseSource,
+	return signature === false
+		? processInputs(allVotes)({ passphrase: null, secondPassphrase: null })
```

Comment:
Since `processInputs(allVotes)` is common to both branches, it might be worth extracting it into a variable:
```
const allVotes = ...
const processFunction = processInputs(allVotes)

return signature === false
    ? processFunction({ passphrase: null, secondPassphrase: null })
    : getInputsFromSources(...)
        .then(processFunction)
```
(Also applies to some of the other files.)